taxi/views.py

```diff
@@ -36,6 +45,24 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs) -> dict:
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
+        name = self.request.GET.get("name")
```

Comment:
If there is no `name`, it will return `None`. So we should provide here a default value

```suggestion
        name = self.request.GET.get("name", "")
```

Fix in all cases